### ISSUE TITLE
Change headers in bulk upload instructions to `<p>`

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5135]
## Area or Component
Bulk results upload

## Quick Summary
The list instructions for the bulk upload are headers when they should be `<p>`.

## Accessibility Violation
[Text should not be marked as a heading](https://cdc-devtools.dequecloud.com/rule-help/en/semantic-heading-misused)

## Action Requested
Change the headers to a `<p>` and maintain the style as needed.

## Screenshots (optional)
![Screen Shot 2023-01-25 at 21.36.48.png](https://images.zenhubusercontent.com/5ffc8b9b4950f50bf5829d88/d3548b58-1c1e-479f-96cd-9648b8d46a7c)

## Source Code (optional)

[which file or line is the issue attached to?]: #
